content/posts/integration-test-code-coverage-in-go/index.md

```diff
@@ -0,0 +1,215 @@
++++
+date = "2021-11-05T17:10:49+07:00"
+author = "ds0nt"
+description = "After we build our microservices to collect code coverage and run integration tests on them, we merge our code coverage from the kubernetes containers to get a final coverage figure"
+title = "Test Coverage of Go Services during Integration Tests"
+categories = ["DevSecOps", "Testing"]
+tags = ["Kubernetes", "Docker", "integration-test", "Golang", "microservices", "code-coverage]
+slug = "integration-test-code-coverage-in-go"
++++
+# Test Coverage of Go Services during Integration Tests
```

Comment:
I think we should introduce a complex case where these technique help,
For example in our case where we have a HTTP API together with a daemon running in background.

Reply:
Maybe it's more clear now... just having multiple services seems like a good reason, right?